### DISCUSSION TITLE
the gzip file actions-runner-linux-x64-2.306.0.tar.gz does not come down as gzip format

### DISCUSSION THREAD

**Discussion by gjew168** [community/community#61745] [Actions]
### Select Topic Area

Bug

### Body

Downloading the gzip file and trying to extract the file creates an error.  The file format comes down as either UTF-8 or ASCII and not gzip.  This happens as well when going through our self-hosted GHE instance.

curl -O -L https://github.com/actions/runner/releases/download/2.306.0/actions-runner-linux-x64-2.306.0.tar.gz --insecure
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     26      0 --:--:-- --:--:-- --:--:--    27
C02G46DXMD6R:github-actions-runner-base gjew$ tar xvf actions-runner-linux-x64-2.306.0.tar.gz 
tar: Error opening archive: Unrecognized archive format
C02G46DXMD6R:github-actions-runner-base gjew$ file actions-runner-linux-x64-2.306.0.tar.gz 
actions-runner-linux-x64-2.306.0.tar.gz: ASCII text, with no line terminators

**Comment by moonfury-ops** [comment]
Thank you
